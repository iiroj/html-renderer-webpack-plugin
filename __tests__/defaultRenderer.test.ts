import { RendererArgs, defaultRenderer } from "../index";

const assets = { js: ["5749aae7.js", "2ff13846.js"] };
const publicPath = "/";

describe("defaultRenderer", () => {
  it("should generate html", async () => {
    const html = await defaultRenderer({ assets, publicPath });
    expect(html).toMatchSnapshot();
  });
});
