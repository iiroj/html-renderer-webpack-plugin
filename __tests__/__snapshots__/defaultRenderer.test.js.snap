// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultRenderer should generate html 1`] = `
"<!doctype html><head><meta charset=\\"utf-8\\"><title>HtmlRendererWebpackPlugin</title><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\"></head><div id=\\"root\\"></div><script defer type=\\"text/javascript\\" src=\\"/5749aae7.js\\"></script>
<script defer type=\\"text/javascript\\" src=\\"/2ff13846.js\\"></script>"
`;
